src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { updateTodo } from '../../Api/todos';
+
+interface Props {
+  todo: Todo,
+  onDelete: (id: number) => void,
+  changeStatus: (id: number, property: Partial<Todo>) => void,
+}
+
+export const TodoItem: FC<Props> = ({ todo, onDelete, changeStatus }) => {
+  const {
+    title,
+    completed,
+    id,
+  } = todo;
+  const [todoTitle, setTitle] = useState(title);
+  const [isEditing, setIsEditing] = useState(false);
+
+  const submitEditing = () => {
+    if (!todoTitle) {
+      onDelete(id);
+    } else {
+      updateTodo(id, { title: todoTitle });
```

Comment:
After updating the todo, call `changeStatus` to update the local state with the new title.